CONTRIBUTING.md

```diff
@@ -0,0 +1,211 @@
+# Contributing to dosbox-staging
+
+Thank you for your interest in contributing to dosbox-staging! There are many
+ways to contribute, and we appreciate all of them. This document is a bit long,
+so here's links to the major sections:
+
```

Comment:
Suggest some high level categories after the intro block that can carry people down the different pathways.
 - [ 1 Documentation & Support]
 - [ 2 Testing & Issue Reporting]
 - [ 3 Bug Fixing]
 - [4 Code / Feature Contribution]

(even if we leave some of these as TODOs, atleast we've got the major categories here.  Then we can

Reply:
An example of those high-level sections, here: https://www.debian.org/intro/help
(although they breakout each categories content fully into new pages; I don't we have to.)